packages/application-shell/src/apollo-links/utils.ts

```diff
@@ -60,7 +60,11 @@ const isGraphQLError = (
   error: ErrorResponse['graphQLErrors']
 ): error is GraphQLError[] =>
   Array.isArray(error) &&
-  error.some((err) => (err as GraphQLError).extensions !== undefined);
+  error.some(
+    (err) =>
+      (err as GraphQLError).message !== undefined ||
+      (err as GraphQLError).extensions !== undefined
```

Comment:
I think message is even required on any `GraphQLError` according to the type. Maybe it alone is enough. Open for a discussion.